package.json

```diff
@@ -35,7 +39,9 @@
     "banana-i18n": "^2.3.3",
     "better-docs": "^2.7.2",
     "bootstrap": "^4.6.1",
+    "bootstrap-input-spinner": "^3.1.10",
     "bootstrap-tagsinput": "^0.7.1",
+    "canvas": "^2.9.1",
```

Comment:
this package seems to have additional dependencies like cairo, we may have to update documentation

Reply:
Sure, I will discuss this with the mentor on call 👍🏽